repose-aggregator/core/repose-core/src/main/java/org/openrepose/core/services/deploy/ContainerConfigurationListener.java

```diff
@@ -56,14 +56,11 @@ public synchronized void configurationUpdated(ContainerConfiguration configurati
                 if (ad.getCheckInterval() > 0) {
                     dirWatcher.updateCheckInterval(ad.getCheckInterval());
                 }
-
-                dirWatcher.updateArtifactDirectoryLocation(new File(ad.getValue()));
-
-                deploymentDirectory = new File(dd.getValue());
+                dirWatcher.updateArtifactDirectoryLocation(new File(ad.getValue().trim()));
+                deploymentDirectory = new File(dd.getValue().trim());
```

Comment:
I guess you are correct that it doesn't have to be a fully qualified path, but it should be. We should probably update the docs to say that. The reason being, you can technically start the JVM from any directory as long as all the right arguments are provided. This is not the typical scenario, so some behavior may be assumed, but you know how that old saying goes.

And if the only thing we are preventing someone from doing is starting or ending a file name with a space, then I'm pretty ok with that.